VS_Project/Pokerapp/Pokerapp/Tableview.cshtml

```diff
@@ -38,7 +38,7 @@
         int new_Round = current_Round + 1;
         db.Execute("UPDATE Room SET Blind_Round = @0 WHERE Room_Code = @1", new_Round, roomCode);
         <script>
-            $(document).ready(function() {
+            $(document).ready(function () {
```

Comment:
Weet niet waarom maar die spatie triggered mij

Reply:
Fuck you